composer.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
Add corresponding composer.lock file as well, which should have been generated. Also please add steps in readme for others how they need to get the composer working on their local. Like after taking the pull they need to run composer install

Reply:
mmm I cant find any composer.lock file generated in my root
I guess because I haven't used any external library here as there is nothing to do with it!
AND yeah I'll composer thing in the readme

Reply:
@guptahemant  take a look so that I can finalize this PR asap

Reply:
Ok if its not generated then you can't add it.